### ISSUE TITLE
Manual User Login

### ISSUE THREAD

**Issue by xoxys** [Antergos/lightdm-webkit2-greeter#44]
Hi,
is there a way to enable manual user login? The option in lightdm.conf
greeter-show-manual-login = true 
seems not to work...

Thanks for your help

**Comment by lots0logs** [comment]
No, I'm afraid manual login is not currently supported in the default theme. Sorry!

**Comment by xoxys** [comment]
Bad... So to use Antergos with LDAP/Active Directory there is only the ugly "simple theme" or GTK-Greeter?

**Comment by lots0logs** [comment]
Yes, I'm afraid so.

**Comment by xoxys** [comment]
There was an request last year https://github.com/Antergos/lightdm-webkit2-greeter/issues/7

Is there a way to implement this feature in near future?

**Comment by lots0logs** [comment]
There is a chance, yes. Though I can't promise when it will be done. My schedule is pretty full at the moment. Sorry!

**Comment by xoxys** [comment]
No Problem and thank you for your work.

**Comment by rda0** [comment]
@xoxys I made a theme [luminosity](https://github.com/rda0/web-greeter-theme-luminosity) (based on [this theme](https://github.com/muhammadsayuti/lightdm-webkit-theme-luminos)) suitable for manual user login (LDAP/Kerberos/AD). It also presents some static information, customizable via a json config file.

@lots0logs I could not find documentation about callback functions in the [API Documentation](https://doclets.io/Antergos/web-greeter/stable), but I found references in the [manpage](https://github.com/Antergos/web-greeter/blob/master/dist/web-greeter.1). However in the manpage the function to start a session is specified as `lightdm.start_session_sync(session)` whereas the API Documentation states `lightdm.start_session(session)`. What is the difference?

I only tested this with web-greeter version 2.2.5. Are there any changes to the API with the new 3.0.0 beta? Do you have a changelog?
